src/common/constants.ts

```diff
@@ -116,7 +116,7 @@ export const is24HourWindowOver = (time: any) =>
 // connection retry attempt configuration
 export const CONNECTION_RECONNECT_ATTEMPTS = 5;
 
-export const MEDIA_MESSAGE_TYPES = ['IMAGE', 'AUDIO', 'VIDEO', 'DOCUMENT', 'STICKER'];
+export const MEDIA_MESSAGE_TYPES = ['IMAGE', 'VIDEO', 'DOCUMENT', 'STICKER'];
```

Comment:
I think this is used in multiple places like adding attachment and interactive messages. Please check if this is not creating an issue in other places.

Reply:
I fixed this in the [Commit](https://github.com/glific/glific-frontend/pull/2089/commits/f9d5009d5e181c43d6290f5ac83df3aef624dc8e)